### PR TITLE
Initial commit Microsoft.Win32.Registry.Tests

### PR DESCRIPTION
**Pull request by Priya91**

### CODE REVIEW

File: src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{7F5F5134-00FE-4DE8-B20C-3DA8BA2EBA68}</ProjectGuid>
```
Comment (krwq) [inline src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj]: Is the GUID unique or a copy & paste from another project?

File: src/Microsoft.Win32.Registry/tests/Registry/Registry_Getvalue_str_str_obj.cs [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        [Fact]
+        public void TestGetValue01()
+        {
+            // [] Null arguments should throw
+            Action a = () => { Microsoft.Win32.Registry.GetValue(null, null, null); };
+            Assert.Throws<ArgumentNullException>(() => { a(); });
+        }
+
+        [Fact]
+        public void Test02()
```
Comment (krwq) [inline src/Microsoft.Win32.Registry/tests/Registry/Registry_Getvalue_str_str_obj.cs]: Please create an issue to clean up tests names and remove those comments

**Comment by mmitche** [comment]
@Priya91 Build will fail initially because there are no tests to run.

**Comment by Priya91** [comment]
@mmitche : How do I fix this ?

**Comment by mmitche** [comment]
@Priya91 not totally sure, but it appears that the dll containing the built tests appears to have no annotated tests according to xunit:

16:25:50 EXEC : error : Microsoft.Win32.Registry.Tests has no tests to run [C:\Jenkins\workspace\dotnet_corefx_windows_prtest\src\dirs.proj]
16:25:50 C:\Jenkins\workspace\dotnet_corefx_windows_prtest\src\packages\Microsoft.DotNet.BuildTools.1.0.20-prerelease\lib\tests.targets(41,5): error MSB3073: The command "corerun.exe xunit.console.netcore.exe  "C:\Jenkins\workspace\dotnet_corefx_windows_prtest\bin\tests\Microsoft.Win32.Registry.Tests\Debug\aspnetcore50\Microsoft.Win32.Registry.Tests.dll" -notrait "category=failing" " exited with code -1. [C:\Jenkins\workspace\dotnet_corefx_windows_prtest\src\dirs.proj]

**Pull request merged**